engine/docs/src/script_doc.py

```diff
@@ -405,7 +473,6 @@ def parse_document(doc_str):
     if doc_info:
         doc.info.CopyFrom(doc_info)
 
-    element_list.sort(cmp = lambda x,y: cmp(x.name, y.name))
```

Comment:
Better leave any sorting to the doc generation.
In C++ headers, the functions generally already come in a "sorted" fashion, which we'd like to keep.